vtex/toolbelt
# Refact/plugin redirects

#### What is the purpose of this pull request?
Transform redirects command into plugin

#### What problem is this solving?

#### How should this be manually tested?
Run all redirects subcommands and see if the behavior is as expected
- `vtex-test redirects export test.csv`
- `vtex-test redirects delete test.csv`
- `vtex-test redirects import test.csv`

#### Screenshots or example usage
<img width="526" alt="Screen Shot 2020-08-07 at 12 26 14" src="https://user-images.githubusercontent.com/19983991/89661533-32b4be00-d8a9-11ea-923d-48cfdf1a8f52.png">

#### Types of changes

#### Chores checklist
